docs/vsphere-golden-image.md

```diff
@@ -147,6 +147,8 @@ makes it ready for future deployments, maintaining all the changes needed for th
 You can find an example [unattend.xml](unattend.xml) file in this repository. It is just an example and, it **should 
 not** be used directly. You must customize the example to fit your needs.
 
+Before executing the `sysprep` command, clone the VM and then run `sysprep` command in the cloned VM.
```

Comment:
We need to call out the hostname requirements somewhere in these steps.

Reply:
You can find it [here](https://github.com/openshift/windows-machine-config-operator/pull/515/commits/3f983dffb1af3d1d8c2e35e70d8b1b09b1f20cb2#diff-eff7340e15df41ff3a96507abf744eb5fea3f670fbb381edc624b52cda6b1b79R150), it was added as part of this [commit](https://github.com/openshift/windows-machine-config-operator/pull/515/commits/68fac906e2a151b4c049efd41b0db29c4e441695#diff-eff7340e15df41ff3a96507abf744eb5fea3f670fbb381edc624b52cda6b1b79R150) where t[he computer name is introduced](https://github.com/openshift/windows-machine-config-operator/pull/515/commits/68fac906e2a151b4c049efd41b0db29c4e441695#diff-f95d5d0ee98dedf392fb54c0fb93f7e3df3c9699b01d08ca8fd00d30ccba8244R71)

Reply:
None of these links work anymore which is expected when you force push a commit.

Reply:
You have added docs for this in the CI steps but not in [vsphere-golden-image.md](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/vsphere-golden-image.md)

Reply:
My bad, links updated. PTAL